app/src/main/java/com/example/lawn_care/viewYourProperties.java

```diff
@@ -97,10 +106,28 @@ public void onResponse(String response) {
                                     //you can change the textview settings, for example
                                     TV_address.setTextSize(20);
                                     //IDEA: try changing some other attributes to make it look better
```

Comment:
maybe add more changes to the listings to make more readable? or handle long addresses to prevent them from going over multiple lines, have ellipse at end of string if its too long, use ellipsize